README.md

```diff
@@ -9,78 +9,39 @@
+- as well as other [blog posts about OQT](https://heigit.org/tag/oqt-en)
 
 
-### API
-
-Head over to [https://oqt.ohsome.org/api/docs](https://oqt.ohsome.org/api/docs) to explore the OQT API interactively.
-
-The response schema is defined here: [docs/api.md](/docs/api.md)
-
+## OQT Components
 
-### CLI
+This diagram gives an overview of the OQT components:
 
-To use the OQT CLI the `ohsome_quality_analyst` Python package needs to be installed and configured. Additionally access to the OQT database is required. Please reach out if you need one.
+![](/docs/img/UML-Component-Diagram.png)
 
-Python 3.8 is required.
 
-```bash
-$ cd workers
-$ pip install .
-$ # Configure OQT
-$ oqt --help
-Usage: oqt [OPTIONS] COMMAND [ARGS]...
+## Usage
 
-Options:
-  --version    Show the version and exit.
-  -q, --quiet  Disable logging.
-  --help       Show this message and exit.
+OQT can be used in three ways with various degree of usability and flexibility:
+1. Interactive access through the *website* (Highest usability; Lowest flexibility)
+2. Making requests to the *REST API*
+3. Usage of the Command-Line-Interface (*CLI*) (Lowest usability; Highest flexibility)
 
-Commands:
-  create-all-indicators  Create all indicators for a specified dataset.
-  create-indicator       Create an Indicator and print results to stdout.
-  create-report          Create a Report and print results to stdout.
-  list-datasets          List available datasets.
-  list-indicators        List available indicators and their metadata.
-  list-layers            List available layers and how they are definied...
-  list-regions           List available regions.
-  list-reports           List available reports and their metadata.
-```
+The easiest access to quality reports is through the OQT [website](https://oqt.ohsome.org). There reports with pre-computed indicators can be accessed for a selection of regions made by the OQT team. But it is not possible to upload custom areas-of-interest (AOIs) and compute report or indicators for those.
 
+Requests for individual indicators as well as reports can be made to the OQT [API](https://oqt.ohsome.org/api/docs). Retrievable are pre-computed indicators for selected regions (Parameters `dataset=regions` and `featureId`) as well as for custom AOIs (Parameter `bpolys`). The major drawbag is the limitation of the input geometry size. For more information check out the interactive API [documentation](https://oqt.ohsome.org/api/docs).
 
-## OQT Components
+The last way to use OQT is by setting up OQT on your local machine and use the provided CLI. This is the most flexible way but some technical knowledge is required to set it up. For now please refer to the development setup [documentation](/docs/development_setup.md) on how to setup OQT localy on your machine. After installation of the `ohsome_quality_analyst` Python package run `oqt --help` for documation about the CLI.
```

Comment:
The development setup does not cover how you can setup the CLI as the README did.

Reply:
Yes and no. The problem here is that for the CLI to work one must set up a the database and configure OQT accordingly. Currently all those information's are found in the dev docs. 

The CLI set up described here in the README was how to install OQT CLI in theory but a lot of details for getting it to run were missing.